src/main/java/edu/mit/puzzle/cube/core/model/HuntStatusStore.java

```diff
@@ -327,6 +329,44 @@ public boolean setTeamProperty(
         return updates > 0;
     }
 
+    public <P extends Team.Property> boolean mutateTeamProperty(
```

Comment:
How concerned should we be with thread safety here? I think this is the first really non-idempotent change we have going on.